src/popup/components/AccountInfo.vue

```diff
@@ -86,9 +87,9 @@ export default defineComponent({
   },
   setup(props) {
     const store = useStore();
-    const getDefaultName = store.getters['names/getDefault'] as (a?: string) => string;
+    const { getName } = useAeNames({ store });
 
-    const name = computed(() => getDefaultName(props.account.address));
+    const name = computed(() => getName(props.account.address!).value);
```

Comment:
Probably there is no need to add `value` as it will be automatically unwrapped by parent computed.